src/pages/BuySell/GemBuySell.tsx

```diff
@@ -0,0 +1,189 @@
+import { ChevronDownIcon } from '@chakra-ui/icons'
+import {
+  Avatar,
+  Box,
+  Button,
+  Flex,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  useColorModeValue
+} from '@chakra-ui/react'
+import { Text as ChakraText } from '@chakra-ui/react'
+import { getConfig } from 'config'
+import { useLocation } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Page } from 'components/Layout/Page'
+import { Text } from 'components/Text'
+
+export const GemBuySell = () => {
+  const location: any = useLocation()
```

Comment:
type and parse and do this nicer. is the coin list from the state the only way to get this? seems off.